docs/content/preview/yugabyte-platform/back-up-restore-universes/configure-backup-storage.md

```diff
@@ -108,13 +122,17 @@ You can configure Azure as your backup target as follows:
 
     <br/>
 
-    ![Azure Shared Access Signature page](/images/yp/cloud-provider-configuration-backup-azure-generate-token.png)
+    ![Azure Shared Access Signature page](/images/yp/cloud-provider-configuration-backup-azure-generate-token.png)<br><br>
 
 1. On your YugabyteDB Anywhere instance, provide the container URL and SAS token for creating a backup, as follows:
 
     <br/>
 
-    * Navigate to **Configs** > **Backup** > **Azure**.
-    * Enter values for the **Container URL** and **SAS Token** fields, as shown in the following illustration, and then click **Save**.<br><br>
+    * Navigate to **Configs** > **Backup** > **Azure Storage**.
+    * Click **Create AZ Backup** to access the configuration form shown in the following illustration:<br><br><br>
 
-    ![Azure Backup](/images/yp/cloud-provider-configuration-backup-azure.png)
+    ![GCS Cloud Provider Configuration](/images/yp/cloud-provider-configuration-backup-azure.png)<br><br>
+    
+    * Use the **Configuration Name** field to provide a meaning name for you backup configuration.
```

Comment:
```suggestion
    * Use the **Configuration Name** field to provide a meaningful name for your backup configuration.
```

Reply:
LOLOLOL, I copied that "meaning name for you backup" five times